js/bundle.go

```diff
@@ -310,7 +299,12 @@ func (b *Bundle) instantiate(vuImpl *moduleVUImpl, vuID uint64) (moduleInstance,
 		}
 		return nil, err
 	}
-	if exports := instance.exports(); exports == nil {
+	if common.IsNullish(exportsV) {
+		return nil, errors.New("exports must be not nil")
```

Comment:
Is it an end-user message? 

If so, I'd suggest making it something like `exports must be defined` or avoiding using go specific's nil.

Reply:
Slightly different message in 039553ea4

Reply:
very javascriptish to me :+1: